src/SFA.DAS.ApplyService.InternalApi/Controllers/OrganisationSummaryController.cs

```diff
@@ -0,0 +1,296 @@
+
+            return Ok(peopleInControl);
+        }
+
+
+        [HttpGet]
+        [Route(("TrusteeData/CharityCommission/{applicationId}"))]
+        public async Task<IActionResult> GetTrusteesFromCharityCommission(Guid applicationId)
+        {
+            _logger.LogInformation($"Retrieving Apply Data - charity commission - trustees for application {applicationId}");
+            var peopleInControl = new List<PersonInControl>();
+
+            var applyData = await _applyRepository.GetApplyData(applicationId);
+
+            if (applyData?.GatewayReviewDetails?.CharityCommissionDetails == null)
+            {
+                return Ok(peopleInControl);
+            }
+
+            var charityCommissionData = applyData.GatewayReviewDetails.CharityCommissionDetails;
+            if (charityCommissionData?.Trustees == null || !charityCommissionData.Trustees.Any())
+                return Ok(peopleInControl);
+
+            foreach (var pic in charityCommissionData.Trustees.OrderBy(x => x.Name))
+            {
+                var picName = pic?.Name;
+                peopleInControl.Add(new PersonInControl { Name = picName, DateOfBirth = null });
+            }
+
+            return Ok(peopleInControl);
+        }
+
+        [HttpGet]
+        [Route(("WhosInControlData/Submitted/{applicationId}"))]
+        public async Task<IActionResult> GetWhosInControlFromSubmitted(Guid applicationId)
+        {
+            _logger.LogInformation($"Retrieving submitted who's in control for application {applicationId}");
+            var peopleInControl = new List<PersonInControl>();
+
+            _logger.LogDebug($"Retrieving from [AddPeopleInControl] for submitted who's in control for application {applicationId}");
+            var pics =
+                await _qnaApiClient.GetTabularDataByTag(applicationId, RoatpWorkflowQuestionTags.AddPeopleInControl);
+
+            if (pics == null)
+            {
+                _logger.LogDebug($"Retrieving from [AddPartners] for submitted who's in control for application {applicationId}");
+                pics = await _qnaApiClient.GetTabularDataByTag(applicationId, RoatpWorkflowQuestionTags.AddPartners);
+            }
+
+
+            if (pics != null)
+            {
+                _logger.LogDebug($"Constructing list of who's in control from retrieved details for application {applicationId}");
+                if (pics?.DataRows == null || !pics.DataRows.Any()) return Ok(peopleInControl);
+
+                foreach (var pic in pics.DataRows.Where(x => x.Columns.Any()).OrderBy(x => x.Columns[0]))
+                {
+                    var picName = pic.Columns[0];
+                    var picDob = string.Empty;
+                    if (pic.Columns.Any() && pic.Columns.Count >= 2)
+                    {
+                        picDob = pic.Columns[1];
+                    }
+
+                    peopleInControl.Add(new PersonInControl { Name = picName, DateOfBirth = picDob });
+                }
```

Comment:
Can the contents of this if be extracted out into a private? Same for the else block below.